src/ResourceManager/Network/Commands.Network/Cortex/P2sVpnGateway/UpdateAzureRmP2sVpnGatewayCommand.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace Microsoft.Azure.Commands.Network
+{
+    using AutoMapper;
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using System.Security;
+    using Microsoft.Azure.Commands.Network.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+    using Microsoft.Azure.Management.Network;
+    using Microsoft.WindowsAzure.Commands.Common;
+    using MNM = Microsoft.Azure.Management.Network.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+    using System.Linq;
+
+    [Cmdlet("Update",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "P2SVpnGateway",
+        DefaultParameterSetName = CortexParameterSetNames.ByP2SVpnGatewayName,
+        SupportsShouldProcess = true),
+        OutputType(typeof(PSP2SVpnGateway))]
+    public class UpdateAzureRmP2SVpnGatewayCommand : P2SVpnGatewayBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The resource group name.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public virtual string ResourceGroupName { get; set; }
```

Comment:
Both ResourceGroupName and Name should have ParameterSetName = CortexParameterSetNames.ByP2SVpnGatewayName

Reply:
Done.